src/next-app/pages/quote-request.tsx

```diff
@@ -0,0 +1,328 @@
+        console.log(checkedShops);
+        const newCheckedShops = { ...checkedShops, [shop]: false };
+        setCheckedShops(newCheckedShops);
+        handleShopsRefresh(shop, newCheckedShops);
+    }
+
+    const handleShopsSubmit = () => {
+        setOpen(false);
+        const checkedShopsDisplay = [];
+        for (const shop in checkedShops) {
+            if (checkedShops[shop]) {
+                checkedShopsDisplay.push(
+                    <div className={styles['shop-card']} key={shop}>
+                        <div className={styles['shop-title-container']}>
+                            <label key={shop}>
+                                {shop}
+                            </label>
+                        </div>
+                        <div className={styles['remove-button-container']}>
+                            <button className={styles['remove-button']} onClick={() => { handleRemoveShop(shop, checkedShops) }}>Remove</button>
+                        </div>
+                    </div>
+                )
+            }
+        }
+        console.log(checkedShopsDisplay)
+        setCheckedShopsDisplay(checkedShopsDisplay);
+    }
+
+    const handleShopsRefresh = (removedShop: string, newCheckedShops: checkedShops) => {
+        const checkedShopsDisplay = [];
+        console.log(newCheckedShops);
+
+        for (const shop in newCheckedShops) {
+            if (newCheckedShops[shop] && removedShop !== shop) {
+                checkedShopsDisplay.push(
+                    <div className={styles['shop-card']} key={shop}>
+                        <div className={styles['shop-title-container']}>
+                            <label key={shop}>
+                                {shop}
+                            </label>
+                        </div>
+                        <div className={styles['remove-button-container']}>
+                            <button className={styles['remove-button']} onClick={() => { handleRemoveShop(shop, checkedShops) }}>Remove</button>
+                        </div>
+                    </div>
+                )
+                console.log(checkedShopsDisplay);
+            }
+        }
+        console.log(checkedShopsDisplay);
+        setCheckedShopsDisplay(checkedShopsDisplay);
+    }
+
+    return (
+        <div>
+            <Header goBackToHref="/login" title="Create Quote Request" />
+            <div className={styles.subtitle}>
+                <label>
+                    Vehicle Information
+                </label>
+            </div>
+            <div className={styles.content}>
+                <div className={styles['text-container']}>
+                    <TextField name="Quote Request Name" placeholder="Enter a quote request name" onChange={setName} required />
+                </div>
+                <div className={styles['text-container']}>
+                    <SingleDropdownField name="Select Vehicle Make" items={makes} onChange={setMake} required />

```

Comment:
This field should also have a placeholder. "Enter vehicle make..." would make it consistent with the dropdown in the Create Shop screen.